ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts.py

```diff
@@ -1670,13 +1670,15 @@ def underlay(self):
         bgp = self.bgp and self.underlay_routing_protocol == "ebgp"
         mpls = self.underlay_routing_protocol in ["isis-sr", "isis-ldp", "isis-sr-ldp", "ospf-ldp"] and self.mpls_lsr
         ldp = self.underlay_routing_protocol in ["isis-ldp", "isis-sr-ldp", "ospf-ldp"] and mpls
+        sr = self.underlay_routing_protocol in ["isis-sr", "isis-sr-ldp"] and mpls
```

Comment:
you should also set `sr` in line 1669

Reply:
ok I will add `ldp` as well then - did not add SR as it was not there